src/Eventso.Subscription/PooledList.cs

```diff
@@ -8,7 +8,7 @@
 namespace Eventso.Subscription
 {
     [DebuggerDisplay("Count = {Count}")]
-    internal class PooledList<T> : IMemoryOwner<T>, IConvertibleCollection<T>
+    public class PooledList<T> : IMemoryOwner<T>, IConvertibleCollection<T>
```

Comment:
bad decision, you can  use internalsvisibleto

Reply:
done